---
title: "Module 3: Project 2 by Team 5"
subtitle: "Charting the distributed nitrogen cycle"
author: 
  - Karl Abuan
  - May Ho
  - Jonah Lin
  - Leilynaz Malekafzali
  - Tiffany Yang
  - Abdur Rahman M. A. Basher
abstract: |
  This is the abstract.
  It consists of two paragraphs.
date: "`r format(Sys.time(), '%B %d, %Y')`"
header-includes:
   - \newcommand*{\secref}[1]{Section~\ref{#1}}
output:
  html_document:
    toc: yes
    toc_float:
      collapsed: no
  pdf_document:
    toc: yes
    toc_depth: 3
    number_sections: yes
    fig_caption: true
mainfont: Times
fontsize: 11 pt
editor_options:
  chunk_output_type: console
urlcolor: blue
link-citations: yes
bibliography: main.bib
---

```{r global_options, include=FALSE}
knitr::clean_cache(clean = TRUE)
knitr::opts_chunk$set(fig.width=8, fig.height=6, fig.align = "center", echo=FALSE, warning=FALSE, include=TRUE, tidy = TRUE, message=FALSE)
```


```{r, eval=FALSE}
## try http:// if https:// URLs are not supported
source("https://bioconductor.org/biocLite.R")  
biocLite("phyloseq")
library("tidyverse")
library("gridExtra")
library("magrittr")
library("ggpubr")
library("cowplot")
```

```{r, include=FALSE}
source("https://bioconductor.org/biocLite.R")  
library("phyloseq")
library("tidyverse")
library("gridExtra")
library("magrittr")
library("ggpubr")
library("knitr")
library("cowplot")

theme_set(theme_bw())
pal = "Set1"
scale_colour_discrete <-  function(palname=pal, ...){
  scale_colour_brewer(palette=palname, ...)
}
scale_fill_discrete <-  function(palname=pal, ...){
  scale_fill_brewer(palette=palname, ...)
}
```

* * *
# Abstract

# Introduction

Saanich Inlet, located in southeastern Vancouver Island, is an intermittently anoxic fjord, and therefore an ideal model to seasonally study microbial community responses to ocean deoxygenation (1-2). Established by the shallow sill opening to the Strait of Georgia, circulation of basin water in Saanich Inlet is reduced. Together with poor ventilation and intense downward flux of organic matter by aerobically respiring organisms living on surface waters (3), oxygen levels down the inlet water column decreases followed by changes in nutrient availability. In the lack of oxygen, microbes turn to use alternative terminal electron acceptors such as nitrate, followed by hydrogen sulfide which is developed at the bottom of the water column (4). Thus, these oxygen-deprived regions, termed as oxygen minimum zones (OMZs), post a diverse microbial community that mediate important biogeochemical processes such the nitrogen cycle. Since OMZs provide appropriate conditions to enable substantial nitrogen loss, we aim to reconstruct the nitrogen cycle in Saanich Inlet to study the microbial community responses to varying levels of oxygen deficiency with specific emphasis on the denitrification pathway.

**The Nitrogen Cycle**

Nitrogen is the fourth most abundant element in the cellular biomass (5). Microbial activities entirely control the interchange between the inert N2 gas in the atmosphere to usable nitrogen that can support cellular metabolism and growth (5). The nitrogen cycle consists of four significant nitrogen-transformation flows: ammonification, nitrification, denitrification, anammox (5). Ammonification, includes nitrogen fixation, where N2 is reduced to ammonium by bacteria and archaea that encode nitrogenase enzyme, and also the assimilatory and dissimilatory reduction of nitrate to ammonium (5). It should be noted that nitrogen fixation is an oxygen sensitive process as nitrogenase enzyme is inhibited by oxygen (6). Nitrification is the process of oxidation of ammonia to nitrite, and further oxidation of nitrite to nitrate (5). Denitrification is the process of dissimilatory reduction of nitrate (NO~3~^-^) and nitrite (NO~2~^-^) to the gaseous oxides, nitric oxide (NO), and nitrous oxide (N~2~O), which may further get reduced to dinitrogen gas (N~2~) (5). In anaerobic ammonium oxidation commonly abbreviated as anammox, the pools of nitrite (N~2~O^-^) and ammonium (NH~4~^+^) are utilized to form N~2~ (5). Anammox is used for wastewater treatment, as it removes both ammonium and nitrite simultaneously without producing N~2~O (5).

The denitrification process post great importance in the global nitrogen balance. It returns the nitrogen into the atmosphere and maintains homeostasis between the nitrogen content of soil and air. (8). The four functional enzyme of denitrification are: nitrate, nitrite, nitric oxide and nitrous oxide reductase which are encoded by *nar*, *nir*, *nor*, and *nos*, respectively (9). Denitrification takes place in aquatic, marine, and terrestrial ecosystems, and the rate of this process is affected by various factors such as pH, temperature, and oxygen concentration (8). In OMZs, nitrate (NO~3~^-^) and nitrite (NO~2~^-^) act as the main terminal electron acceptors in respiration (10). Nitrogen oxide reductases are sensitive to oxygen and it is known that oxygen represses denitrification (8). Among the four N-oxide reductases, nitrous oxide reductase is the most sensitive enzyme to oxygen, leading to the increase in N~2~O production in the suboxic and anoxic conditions (10). The interest in denitrification process is for several reasons. Denitrification is the process contributing to the loss of fertilizer nitrogen, therefore reducing the agriculture productivity (7). Additionally, denitrification can be utilized to remove nitrogen from high-nitrate wastewaters to decrease the contamination and eutrophication (7). However, denitrification can have negative effects on the climate system, as the emission of N~2~O leads to stratospheric ozone layer depletion, contributing to global warming (7). Some of the major denitrifying bacteria are from genera: *Alcaligenes*, *Agrobacterium*, *Bacillus*, *Hyphomicrobium*, *Moraxella*, *Pseudomonas*, and *Paracoccus* (7). Due to the high frequency of isolation, the genera *Pseudomonas* and *Alcaligenes* are of the greatest significance (4).

In this project, our objective is to reassemble the nitrogen cycle as a distributed metabolic process which may provide new insight into the roles of microbes play on nutrient and energy fluxes in the ocean. There are several well-defined gene encoding key steps in the nitrogen cycle and they serve as basis for functional anchor screening to determine their distribution across the Tree of Life. Here, we focused on the nitric oxide reductase, *norB*, and we examined its DNA and RNA abundance across the different depths in Saanich Inlet. We also investigated how the nitrogen species in Saanich relate to the abundances of *norB*.


# Materials and Experimental Configuration
## Experimental Protocols

To understand the correlation of microbial diversity and oxygen concentration across samples, we report four experimentally designed test protocols:

**P1.** Analysis of the DNA abundance of *norB* with depth.

**P2.** Analysis of similarities between RNA and DNA abundance information of *norB* with depths.

**P3.** Reconstructing the associated taxa with *norB* and analyze variances of DNA and RNA based on depths.

**P4.** Analysis of the abundance of *norB* in relation to nitrogen species in Saanich.


## Dataset

```{r loadingDNA}
## DNA
norB.DNA.10m <- read_tsv("data/dna/norB_DNA_10m_marker_contig_map.tsv") %>% 
  select(Tax.DNA.10 = Confident_Taxonomy, Abund.DNA.10 = Abundance, Query)

norB.DNA.100m <- read_tsv("data/dna/norB_DNA_100m_marker_contig_map.tsv") %>% 
  select(Tax.DNA.100 = Confident_Taxonomy, Abund.DNA.100 = Abundance, Query)

norB.DNA.120m <- read_tsv("data/dna/norB_DNA_120m_marker_contig_map.tsv") %>% 
  select(Tax.DNA.120 = Confident_Taxonomy, Abund.DNA.120 = Abundance, Query)

norB.DNA.135m <- read_tsv("data/dna/norB_DNA_135m_marker_contig_map.tsv") %>% 
  select(Tax.DNA.135 = Confident_Taxonomy, Abund.DNA.135 = Abundance, Query)

norB.DNA.150m <- read_tsv("data/dna/norB_DNA_150m_marker_contig_map.tsv") %>% 
  select(Tax.DNA.150 = Confident_Taxonomy, Abund.DNA.150 = Abundance, Query)

norB.DNA.165m <- read_tsv("data/dna/norB_DNA_165m_marker_contig_map.tsv") %>% 
  select(Tax.DNA.165 = Confident_Taxonomy, Abund.DNA.165 = Abundance, Query)

norB.DNA.200m <- read_tsv("data/dna/norB_DNA_200m_marker_contig_map.tsv") %>% 
  select(Tax.DNA.200 = Confident_Taxonomy, Abund.DNA.200 = Abundance, Query)

## RNA
norB.RNA.10m <- read_tsv("data/rna/norB_RNA_10m_marker_contig_map.tsv") %>% 
  select(Tax.RNA.10 = Confident_Taxonomy, Abund.RNA.10 = Abundance, Query)

norB.RNA.100m <- read_tsv("data/rna/norB_RNA_100m_marker_contig_map.tsv") %>% 
  select(Tax.RNA.100 = Confident_Taxonomy, Abund.RNA.100 = Abundance, Query)

norB.RNA.120m <- read_tsv("data/rna/norB_RNA_120m_marker_contig_map.tsv") %>% 
  select(Tax.RNA.120 = Confident_Taxonomy, Abund.RNA.120 = Abundance, Query)

norB.RNA.135m <- read_tsv("data/rna/norB_RNA_135m_marker_contig_map.tsv") %>% 
  select(Tax.RNA.135 = Confident_Taxonomy, Abund.RNA.135 = Abundance, Query)

norB.RNA.150m <- read_tsv("data/rna/norB_RNA_150m_marker_contig_map.tsv") %>% 
  select(Tax.RNA.150 = Confident_Taxonomy, Abund.RNA.150 = Abundance, Query)

norB.RNA.165m <- read_tsv("data/rna/norB_RNA_165m_marker_contig_map.tsv") %>% 
  select(Tax.RNA.165 = Confident_Taxonomy, Abund.RNA.165 = Abundance, Query)

norB.RNA.200m <- read_tsv("data/rna/norB_RNA_200m_marker_contig_map.tsv") %>% 
  select(Tax.RNA.200 = Confident_Taxonomy, Abund.RNA.200 = Abundance, Query)
```

## Data Preporocessing

Manipulate the data into a single data frame

```{r preprocessing}
norB.all <- norB.DNA.10m %>% 
# Combine the data frames will full_join to keep all the data
  full_join(norB.DNA.100m, by = "Query") %>%
  full_join(norB.DNA.120m, by = "Query") %>% 
  full_join(norB.DNA.135m, by = "Query") %>%
  full_join(norB.DNA.150m, by = "Query") %>% 
  full_join(norB.DNA.165m, by = "Query") %>% 
  full_join(norB.DNA.200m, by = "Query") %>% 
  full_join(norB.RNA.10m, by = "Query") %>%
  full_join(norB.RNA.100m, by = "Query") %>% 
  full_join(norB.RNA.120m, by = "Query") %>% 
  full_join(norB.RNA.135m, by = "Query") %>%
  full_join(norB.RNA.150m, by = "Query") %>% 
  full_join(norB.RNA.165m, by = "Query") %>% 
  full_join(norB.RNA.200m, by = "Query") %>% 
# Create a taxonomy variable aggregating all taxonomy columns so as to fill in any NAs that might occur. !is.na means "is not NA", so the following says that the Taxonomy data should be taken from Tax.RNA.10 if that is not NA, or else take it from Tax.DNA.10 if that is not NA, or else Tax.RNA.200, etc. until if all are NA, give Taxonomy of "unclassified"
  mutate(Taxonomy = ifelse(!is.na(Tax.DNA.10), Tax.DNA.10,
                    ifelse(!is.na(Tax.DNA.100), Tax.DNA.100,
                    ifelse(!is.na(Tax.DNA.120), Tax.DNA.120,
                    ifelse(!is.na(Tax.DNA.135), Tax.DNA.135,
                    ifelse(!is.na(Tax.DNA.150), Tax.DNA.150,
                    ifelse(!is.na(Tax.DNA.165), Tax.DNA.165,
                    ifelse(!is.na(Tax.DNA.200), Tax.DNA.200,
                    ifelse(!is.na(Tax.RNA.10), Tax.RNA.10,
                    ifelse(!is.na(Tax.RNA.100), Tax.RNA.100,
                    ifelse(!is.na(Tax.RNA.120), Tax.RNA.120,
                    ifelse(!is.na(Tax.RNA.135), Tax.RNA.135,
                    ifelse(!is.na(Tax.RNA.150), Tax.RNA.150,
                    ifelse(!is.na(Tax.RNA.165), Tax.RNA.165,
                    ifelse(!is.na(Tax.RNA.200), Tax.RNA.200,
                           "unclassified"))))))))))))))) %>% 
# Get rid of the old Tax variables
  select(-starts_with("Tax.")) %>% 
# Gather all the abundance data into 1 column 
  gather("Key", "Abundance", starts_with("Abund")) %>% 
# Turn the Key into Depth and RNA/DNA variables. We can easily do this because we specifically named these variables with period separation when we loaded in the original data
  separate(Key, c("Key","Type","Depth_m"), by = ".") %>% 
# Remove Key variable now that it only contains "Abund". This also serves to reorder the columns so that the very long Query is at the end.
  select(Depth_m, Type, Abundance, Taxonomy, Query) %>% 
# Make sure R knows depth is numerical since it came from a character variable
  mutate(Depth_m = as.numeric(Depth_m)) %>%
  # Make sure to impute missing abundant information by setting it to 0
  mutate(Abundance = ifelse(is.na(Abundance), 0, Abundance)) %>%
  # Separate Taxonomy into columns so we can plot at different levels
  separate(Taxonomy, into = c("Domain", "Phylum", "Class", "Order", "Family", "Genus", "Species"), sep="; ")
```

```{r normalize}
# Also, normalize abundance information relative based on DNA or RNA 
norB.all.norm <- norB.all %>% 
  group_by(Type) %>%
  mutate(Abundance = ifelse(Abundance==0, 0, Abundance/sum(Abundance))) %>%
  ungroup()
```

## Methods

Sequences for norB were obtained from various depths at Saanich Inlet. Listed below are the method of processing the tree-based sensitive and accurate protein profiler (TreeSAPP):  
1)     The query sequences were first mapped to the reference sequences in BLASTp.  
2)     Coding sequences were then extracted and queried with putative functions.  
3)     Profile-alignment (HmmAlign) with multiple alignment of references and query.  
4)     Classification of sequence insertions were then done in reference trees (RAxML).  
5)     Reference trees were then updated with novel sequences.  
6)     Construction of trees at various depths with Interactive Tree of Life (iTOL).  

Majority of TreeSAPP pipeline was run on Google Cloud for metagenomic (MetaG, DNA) and metatranscriptomic (MetaT, RNA) assemblies of norB gene at the depths of 10m, 100m, 120m, 135m, 150m, 165m, and 200m. The following codes below were used in the pipeline:

time treesapp.py -T 8 --verbose --delete --pairing pe \
Various functions were stated with the following line above for: Tracking the runtime, specifying the maximum number of threads (8) to be used for efficiency, printing the intermediate steps, deleting the intermediate files to reduce the storage space occupied, and lastly specifying the usage of paired end files for comparing our data.

-t D0501 \
Target gene was specified here (D0501, which corresponded to norB).

-i bucket/MetaG_assemblies/depth#_assembly.fasta --rpkm \
Input files and path to store files was specified with the following line above.

-r bucket/MetaG_reads/depth#.fastq.gz \
The input files for MetaG and MetaT reads were also specified above.

-o treesapp_out_dir_depth#
Output file name was specified above.

rm treesapp_out_dir_depth1/RPKM_outputs/*.sam
Last instruction here was used to remove gigabyte size intermediate SAM files.

Visualization of output files at various depths were done on iTOL v4 for norB gene.

Using the processed results, DNA abundance and RNA abundance analyses were done for norB at various depths (Fig. 1 and Fig. 2). In addition, various genera were also compared in regards to their DNA vs. RNA abundances to determine the expression of norB for each genus at different depths (Fig. 3 and Fig. 4). Lastly, the relationship of abundance of norB and concentration of several nitrogen species – such as nitrate, nitrogen dioxide, and nitrous oxide – were also observed (Fig. 5).


# Results

## Analysis of the DNA abundance of *norB* with depth \label{sec:community}

```{r}
abun.dna.by.depth <- norB.all %>% 
  filter(Type=="DNA") %>%
  group_by(Depth_m) %>%
  summarise(Abundance_DNA=sum(Abundance))

kable(abun.dna.by.depth,caption="Unnormalized abundance of the norB gene (DNA) at different depths")
```

```{r figure1}
plot1 <- norB.all.norm %>% 
  # Filter the DNA data
  filter(Type == "DNA") %>% 
  # Change NAs to "unclassified" at the level you want to plot. Here we will do Genus
  mutate(Genus = ifelse(is.na(Genus), "unclassified", Genus)) %>% 

  ggplot(aes(x = "norB", y = Depth_m)) +
  # Use the size aesthetic to show abundance
  geom_point(aes(size = ifelse(Abundance == 0, NA, Abundance*100)), colour="brown") +
  # Reverse the why axis so depth increases going down
  scale_y_reverse(lim=c(200,10)) +
  scale_size("Norm. relative abundance (%)") + 
  labs(x = "", y="Depth (m)") +
  theme_classic()

annotate_figure(plot1, bottom = text_grob("Figure 1: Normalized relative abundance (0-1) of the norB gene (DNA) at different depths", size = 12))
```

The DNA abundance of the norB followed an upwards trend where it increased from a normalized relative abundance of 2.5% at depth 100m to 10.0% at depth 200m. This was consistent with what had been shown in previous studies, where genes associated with partial denitrification to nitrous oxide were most abundant at a depth of 150m (11) and were not present at depth 50m. An exception of this trend occurs at depth 165 meters, where the abundance declined sharply. Since the gene encodes the nitric oxide reductase enzyme—responsible for the reduction of nitric oxide to nitrous oxide—the microbes possessing these genes would be considered partial denitrifiers. It is likely that conditions at lower depths have conditions that are more ideal for partial denitrifiers, which would explain why the gene’s presence is more abundance at lower depths. Depth 165m could be less ideal for microbial growth because it sits between the sulfide nitrate transition zone (SNTZ) and the sulfide zone.  

## Analysis of similarities between RNA and DNA abundance information of *norB* with depths \label{sec:OTUabundance}

```{r}
abun.rna.by.depth <- norB.all %>% 
  filter(Type=="RNA") %>%
  group_by(Depth_m) %>%
  summarise(Abundance_RNA=sum(Abundance))

abun.by.depth <- full_join(x = abun.dna.by.depth, 
                           y = abun.rna.by.depth, 
                           by = "Depth_m")
kable(abun.by.depth, caption="Unnormalized abundance of the norB gene (DNA vs. RNA) at different depths")
```

```{r figure2}
plot1<- norB.all.norm %>% 
# Change NAs to "unclassified" at the level you want to plot
  mutate(Genus = ifelse(is.na(Genus), "unclassified", Genus)) %>% 
  # Show both RNA and DNA using an x variable  
  ggplot(aes(x = Type, y = Depth_m)) +
  geom_point(aes(size = ifelse(Abundance == 0, NA, Abundance*100)), colour="brown") +
  scale_y_reverse(lim=c(200, 10)) +
  # Remove zeros by setting min limit to 1e-09
  scale_size("Norm. relative abundance (%)") + 
  labs(x = "", y="Depth (m)") +
  theme_classic()

annotate_figure(plot1, bottom = text_grob("Figure 2: Normalized relative abundance (0-1) of the norB gene (DNA vs. RNA) at different depths", size = 12))
```

The RNA abundance of the norB gene also seemed to follow a similar linear trend like its DNA abundance. It has a normalized relative abundance of 5% at depth 150m and it increased to 15% at depth 200m. We expected this since Hawley *et al.* (12) observed that expressed pathways progressed from ammonia oxidation and nitrification at higher depths to denitrification at lower depths.

## Reconstructing the associated taxa with norB and analyze variances of DNA and RNA based on depths \label{sec:richness}

```{r}
abun.by.phylum <- norB.all %>%
  group_by(Type, Phylum, Genus) %>%
  summarise(Abundance = sum(Abundance)) %>%
  ungroup()

## DNA
abun.dna.by.phylum <- abun.by.phylum %>%
  filter(Type=="DNA")
abun.dna.by.phylum.nonNA <- abun.dna.by.phylum %>%
  filter(!is.na(Genus))

## RNA 
abun.rna.by.phylum <- abun.by.phylum %>%
  filter(Type == "RNA")
abun.rna.by.phylum.nonNA <- abun.rna.by.phylum %>%
  filter(!is.na(Genus))

## Join DNA and RNA
abun.by.phylum <- abun.dna.by.phylum %>% 
  select(Phylum, Genus, Abundance_DNA = Abundance) %>%
  mutate(Abundance_RNA = abun.rna.by.phylum$Abundance)

kable(abun.by.phylum, caption="Taxa associated (including NA values) with unnormalized abundance of the norB gene (DNA vs. RNA) at all depths")
```

The total unnormalized abundances computed from DNA and RNA data is `r sum(abun.dna.by.phylum$Abundance)` and `r sum(abun.rna.by.phylum$Abundance)`, respictavely; however, only `r (sum(abun.dna.by.phylum.nonNA$Abundance))/sum(abun.dna.by.phylum$Abundance) * 100`$\%$ and `r (sum(abun.rna.by.phylum.nonNA$Abundance))/sum(abun.rna.by.phylum$Abundance) * 100`$\%$ have known genera for both DNA and RNA. We impute ``NA`` with ``unclassified`` for phylum and genus. 

<!-- ![Distribution of Species from DNA at 10m depth of Saanich Inlet](final_trees/dna/norB_DNA_10m.pdf){width=60%} -->

<!-- ![Distribution of Species from DNA at 100m depth of Saanich Inlet](final_trees/dna/norB_DNA_100m.pdf){width=60%} -->

<!-- ![Distribution of Species from DNA at 120m depth of Saanich Inlet](final_trees/dna/norB_DNA_120m.pdf){width=60%} -->

<!-- ![Distribution of Species from DNA at 135m depth of Saanich Inlet](final_trees/dna/norB_DNA_135m.pdf){width=60%} -->

<!-- ![Distribution of Species from DNA at 150m depth of Saanich Inlet](final_trees/dna/norB_DNA_150m.pdf){width=60%} -->

<!-- ![Distribution of Species from DNA at 165m depth of Saanich Inlet](final_trees/dna/norB_DNA_165m.pdf){width=60%} -->

<!-- ![Distribution of Species from DNA at 200m depth of Saanich Inlet](final_trees/dna/norB_DNA_200m.pdf){width=60%} -->


```{r figure3}
plot1 <- norB.all.norm %>% 
# Change NAs to "unclassified" at the level you want to plot
  mutate(Phylum = ifelse(is.na(Phylum), "unclassified", Phylum)) %>% 
  
ggplot(aes(x = Phylum, y = Depth_m)) +
# Use an ifelse statement to make 0 values into NAs so that they don't show up on the plot
# Use position_dodge to keep points from overlapping
  geom_point(aes(size = ifelse(Abundance == 0, NA, Abundance*100), color = Type),
             position = position_dodge(0.5)) +
  scale_y_reverse(lim=c(200,10)) +
  scale_size("Norm. relative abundance (%)") + 
  labs(x = "", y="Depth (m)") + 
  theme_classic() +
  theme(axis.text.x = element_text(angle = 45, hjust = 1))

annotate_figure(plot1, bottom = text_grob("Figure 3: Abundance of Phyla with norB (DNA vs. RNA) at different depths", size = 12))
```

According to Table 3, the genus associated with abundance of *norB* includes *Formosa*, *Muricauda*, *Zobellia*, *Achromatium*, *Beggiatoa*, *Candidatus Competibacter*, *Endozoicomonas*, *Gallionella*, *Thiocapsa*, and *Rhizobiales*. These genera belong to three Phylum Bacteriodetes, Chlorobi and Proteobacteria. The abundance of phylum, as shown in Fig. 3, differed with respect to depth and with DNA versus RNA. For Proteobacteria, the abundance of *norB* was greater at 200m in comparison with 100m. At depth of 200m, greater abundance of RNA was observed in comparison with DNA.

```{r figure4}
plot1 <- norB.all.norm %>% 
# Change NAs to "unclassified" at the level you want to plot
  mutate(Genus = ifelse(is.na(Genus), "unclassified", Genus)) %>% 
  
ggplot(aes(x = Genus, y = Depth_m)) +
# Use an ifelse statement to make 0 values into NAs so that they don't show up on the plot
# Use position_dodge to keep points from overlapping
  geom_point(aes(size = ifelse(Abundance == 0, NA, Abundance*100), color = Type),
             position = position_dodge(0.5)) +
  scale_y_reverse(lim=c(200,10)) +
  scale_size("Norm. relative abundance (%)") + 
  labs(x = "", y="Depth (m)") + 
  theme_classic() +
  theme(axis.text.x = element_text(angle = 45, hjust = 1))

annotate_figure(plot1, bottom = text_grob("Figure 4: Abundance of genera with norB (DNA vs. RNA) at different depths", size = 12))
```


## Analysis of the abundance of *norB* in relation to nitrogen species in Saanich \label{sec:ASVabundances}

```{r figure5}
load("data/mothur_phyloseq.RData")
metadata <- data.frame(mothur@sam_data)

plot2 <- metadata %>% 
# Order the data by depth  
  arrange(Depth_m) %>% 
  
ggplot(aes(x = NO3_uM, y = Depth_m)) +
  geom_point(aes(x=NO3_uM, y=Depth_m),size=5, alpha=0.7) +
  geom_path(aes(group = 1)) +
  scale_y_reverse(limits = c(200,10)) +
  labs(y = "Depth (m)", x = "NO3_uM")

plot3 <- metadata %>% 
# Order the data by depth  
  arrange(Depth_m) %>% 
  
ggplot(aes(x = NO2_uM, y = Depth_m)) +
  geom_point(aes(x=NO2_uM, y=Depth_m),size=5, alpha=0.7) +
  geom_path(aes(group = 1)) +
  scale_y_reverse(limits = c(200,10)) +
  labs(y = "Depth (m)", x = "NO2_uM")

plot4 <- metadata %>% 
# Order the data by depth  
  arrange(Depth_m) %>% 
  
ggplot(aes(x = N2O_nM, y = Depth_m)) +
  geom_point(aes(x=N2O_nM, y=Depth_m),size=5, alpha=0.7) +
  geom_path(aes(group = 1)) +
  scale_y_reverse(limits = c(200,10)) +
  labs(y = "Depth (m)", x = "N2O_nM")

annotate_figure(ggarrange(plot2, plot3, plot4, ncol = 3, nrow = 1, 
                                     labels = c("(a)", "(b)", "(c)")),
                bottom = text_grob("Figure 5: Abundance of norB in relation to nitrogen species across depths", size = 12))
```

According to Fig. 5, related to the abundance of NO~3~ and N~2~O, the abundance of *norB* increased from depth 10m to 100m, and declined from 100m to 200m. The abundance of *norB* related to NO~3~ has more of a sharp declined in comparison with that of N~2~O. Regarding to the abundance of *norB* related to NO~2~, a sharp declined is observed from 10m to 100m, the abundance level fluctuated from 100m to 150m, and declined from 150m to 200m.

# Discussion

At the depth of 100m, it appears that most genes involved in the denitrification pathway like *norB* are present, but they are neither abundant nor highly expressed. However, the nir gene encoding nitrite reductase has high abundance and high expression, suggesting that nitrite reductase is actively being reduced to nitric oxide at this depth. The rest of the pathway has low activity by comparison, including the gene encoding the nitrate reductase subunit, napaA, which has low presence and low expression. At the depth of 200m, most genes involved in the denitrification pathway are also presented and are expressed in high level especially for *norB* which is found to be the most abundant in expression level at 200m. Interestingly, *norC* genes is found to have similar expression level compared with *norB* across all depth levels; perhaps those two genes are involved in similar denitrification pathways.

The environmental and evolutionary standpoints are interconnected in regards to the distributed metabolism seen in the nitrogen cycle. At various depths, the temperatures and presence of other chemical species will differ, with some being more significant than others. The differences in temperature and presence of other chemical species may in turn affect the entropy production (13). According to one study (13), biological systems acquire information through evolution and store it within its metagenome for maximum entropy production (MEP). Biological systems may then use this information to achieve MEP and further refine it through evolution. In other words, biological systems that can achieve MEP efficiently are able to allocate resources in expressing the optimal genes for survival in terms of their functional efficiency at the optimal temperatures and resource availability in the environment (13). In terms of the nitrogen cycle, the differences of geochemical gradients at low and high depths – such as the high nitrogen species and low oxygen species at depths 100m and beyond – can be used to explain the increases in abundance of DNA and RNA for norB gene (Fig.2). This observation of oxygen control on denitrifiers was also made in another study (14) that described the decrease and inhibition of denitrification processes at low oxygen concentrations. 

Other studies (11 and 14) have created models to explain the distributed metabolism through gene abundance, geochemical gradients, or both. One model suggested looking at the models separately in terms of MEP and metabolic perspectives (13). In the MEP perspective, the form of the molecular machines is unimportant for biogeochemical MEP. Their model stated that living systems can be viewed as molecular machines that catalyzes reactions that both synthesize and degrade molecular machines and dissipate energy via redox reactions in the process, which are undifferentiated from abiotic systems (13). In the metabolic perspective, their model only predicts the functional characteristics in terms of allocation of molecular machinery (13). In other words, the metabolic model looks at the functional groups and community level gene expression from experiments to explain the distributed metabolism. 

On the other hand, the model proposed by the other study suggested looking at both perspectives in an integrated manner (11). Their model reasoned that both the gene abundance (DNA and RNA) and geochemical concentrations are linked such that gene expression patterns are determined by gene abundances and geochemical conditions with geochemical fluxes being excellent indicators for microbial community structures (11). By looking at the DNA, mRNA, and protein profiles with the geochemical concentrations, different perspectives on the community activity can be obtained in terms of the immediate catalytic potential of a community, the level of expression of control environmental mRNA and protein distributions, and the prediction of pathway expression and growth in relation to geochemical conditions (11). The logics applied to these models and studies can then be used to explain the figure results, which demonstrated how the distributed metabolism in the nitrogen cycle are connected to the gene abundances (Fig. 2) and geochemical gradients (Fig. 4) at various depths.  

---

# References

Introduction Sources:

1. Hawley, AK, Torres-beltrán, M, Zaikova, E, Walsh, DA, Mueller, A, Scofield, M, Kheirandish, S, Payne, C, Pakhomova, L, Bhatia, M, Shevchuk, O, Gies, EA, Fairley, D, Malfatti, SA, Norbeck, AD, Brewer, HM, Pasa-tolic, L, Tijana Glavina Del Rio, Suttle, CA, Tringe, S, Hallam, SJ. 2017. A compendium of multi-omic sequence information from the Saanich Inlet water column. Scientific Data. 4:170160. doi: 10.1038/sdata.2017.160.

2. Hallam, SJ, Torres-beltrán, M, Hawley, AK. 2017. Monitoring microbial responses to ocean deoxygenation in a model oxygen minimum zone. Scientific Data. 4:170158. doi: 10.1038/sdata.2017.158.

3. Ulloa, O, Canfield, DE, DeLong, EF, Letelier, RM, Stewart, FJ. 2012. Microbial oceanography of anoxic oxygen minimum zones. Proc. Natl. Acad. Sci. U. S. A. 109:15996-16003. doi: 10.1073/pnas.1205009109.

4. Wright, JJ, Konwar, KM, Hallam, SJ. 2012. Microbial ecology of expanding oxygen minimum zones. Nature Reviews.Microbiology. 10:381. doi: 10.1038/nrmicro2778.

5. Stein LY, Klotz MG. 2016. The nitrogen cycle . Cell Press 26:R94–R98.

6. Falkowski PG, Fenchel T, Delong EF. 2008. The Microbial Engines That Drive Earth’s Biogeochemical Cycles. Science 320:1034–1039.

7. Knowles R. 1982. Denitrification . Microbiological Reviews 46: 43-70. 

8. Bergaust L, Spanning RJMV, Frostegard A, Bakken LR. 2011. Expression of nitrous oxide reductase in Paracoccus denitrificans is regulated by oxygen and nitric oxide through FnrP and NNR. Microbiology 158:826–834.

9. Dalsgaard T, Stewart FJ, Thamdrup B, Brabandere LD, Revsbech NP, Ulloa O, Canfield DE, Delong EF. 2014. Oxygen at Nanomolar Levels Reversibly Suppresses Process Rates and Gene Expression in Anammox and Denitrification in the Oxygen Minimum Zone off Northern Chile. mBio 5.

10. Qu Zhi, Bakken LR, Molstad L, Frostegard A, Bergaust LL. 2016. Transcriptional and metabolic regulation of denitrification in Paracoccus denitrificans allow low but significant activity of nitrous oxide reductase under oxic conditions. Environmental Microbiology 18: 2951-2963.

Result Sources:

11. Louca S, Hawley AK, Katsev S, Torres-Beltran M, Bhatia MP, Kheirandish S, Michiels CC, Capelle D, Lavik G, Doebeli M, Crowe SA, Hallam SJ. 2016. Integrating biogeochemistry with multiomic sequence information in a model oxygen minimum zone. Proc Natl Acad Sci 113:E5925–E5933.

12. Hawley AK, Brewer HM, Norbeck AD, Pa a-Toli  L, Hallam SJ. 2014. Metaproteomics reveals differential modes of metabolic coupling among ubiquitous oxygen minimum zone microbes. Proc Natl Acad Sci 111:11395–11400.

Discussion Sources:

13. Vallino JJ. 2010. Ecosystem biogeochemistry considered as a distributed metabolic network ordered by maximum entropy production. Philosophical Transactions: Biological Sciences. 365:1417-1427.

14. Christensen S. 1988. Oxygen control prevents denitrifiers and barley plant roots from directly competing for nitrate. FEMS Microbiology Letters. 53:217-221.